solana/trie-geyser-plugin/src/worker.rs

```diff
@@ -128,27 +154,31 @@ impl Worker {
                     if key == slot {
                         break value;
                     }
-                    log::info!("[{key}] dropping accumulator");
+                    log::debug!("#{key}: dropping accumulator");
                 }
                 _ => {
-                    log::error!("[{slot}] accumulator not found");
+                    log::error!("#{slot}: accumulator not found");
                     return;
                 }
             }
         };
 
-        // If the witness account is not in collection of changed accounts,
-        // don’t do anything.
+        // If the trie or witness accounts are not in collection of changed
+        // accounts, don’t do anything.
         if !entry.accounts.contains_key(&self.config.witness_account) {
```

Comment:
Shouldnt we calculate the bank hash when epoch changes?

Reply:
Since we don’t have guest chain there are no epoch changes happening.  With the witness account we only care about the witness account changing.  The only thing it witnesses is the trie and that is only affected by writes to the trie.